### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by Teebusch**
The redesigned page is here: https://teebusch.github.io/uio-carpentry.github.io/
The source is here: https://github.com/Teebusch/uio-carpentry.github.io/tree/use_hugo

Before merging PR:
- [x] use hugo
- [x] use github actions for CI
- [x] render upcoming workshops list
- [x] import governance docs into repository
- [x] import study group content into repository (posts imported, minutes etc left in original repo - see issue in study group repo)
- [x] include reports in repo via module (editing of reports will be done in original repo)
- [x] render reports nicely
- [x] pick and adjust a theme  - use 'book' theme for now. Adjust later, if desired
- [x] proof of concept: add categories, keywords and tags to content

I've moved some points into separate issues for which I think they can be discussed there and solved post-launch

### CODE REVIEW

**Comment by lexnederbragt** [comment]
I really like this design! If others agree, and once @Teebusch is done with the checklist (😉) I suggest we start using it!

File: content/about/charter.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Carpentry@UiO Charter
```
Comment (lexnederbragt) [inline content/about/charter.md:1]: Maybe add a sentence on what is a charter?

File: content/more/code_of_conduct.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+- to report Code of Conduct violations for local Carpentry@UiO
+    workshops, use UiO's
+    [speak-up](https://www.uio.no/english/about/hse/speak-up/index.html)
+    (English) or [si-fra](https://www.uio.no/om/hms/si-fra/) (Norwegian)
+    reporting
+
+If the workshop is a local workshop rather than an official Carpentries
+workshop, the workshop website will have the Carpentry@UiO logo:
+
+![Carpentry@UiO logo](uio-carpentry-logo-small.png)
```
Comment (lexnederbragt) [inline content/more/code_of_conduct.md]: This image does not show on https://teebusch.github.io/uio-carpentry.github.io/more/code_of_conduct/

Comment (Teebusch) [reply content/more/code_of_conduct.md]: Yes, there's an issue with relative image paths in Hugo that I haven't figured out yet.

**Pull request merged**